README.md

```diff
@@ -93,7 +93,7 @@ As a rule of thumb, do the following:
 
 #### Basic formatting
 
-You can use `format()` with constants from the [PhoneNumberFormat](https://github.com/brick/phonenumber/blob/0.2.0/src/PhoneNumberFormat.php) class:
+You can use `format()` with constants from the [PhoneNumberFormat](https://github.com/brick/phonenumber/blob/0.4.0/src/PhoneNumberFormat.php) class:
```

Comment:
You can use "master" instead of version:
https://github.com/brick/phonenumber/blob/master/src/PhoneNumberFormat.php

Reply:
I followed the previous way for links. 
This allows having always valid links - if you will check docs for some old version, links with explicit versions always will be valid.

Anyway, I apply your suggestion.
Let's wait for @BenMorel feedback on the preferred way.

Reply:
Hi, I usually update links just before a release, but it looks like I overlooked this one. Let's use `master` for now, maybe one day I'll have a better way to manage versions in README.